packages/api/lambdas/db-indexer.js

```diff
@@ -130,10 +130,7 @@ function performDelete(esClient, tableIndex, fields, body) {
       id,
       type,
       parent
-    })
-    // Important to catch this error. Uncaught errors will cause
-    // the handler to fail and other records will not be updated.
-    .catch((err) => log.error(err));
```

Comment:
The `catch` is removed here.  Is it because the exception is caught in `indexRecords`?
https://github.com/nasa/cumulus/pull/1030/files#diff-d81ec8a44cfc466b12ce1d6d54f6f0f2L30

Reply:
Correct.